tests/loggers/test_trains.py

```diff
@@ -22,7 +22,7 @@ def test_trains_logger(tmpdir):
     )
     result = trainer.fit(model)
 
-    print('result finished')
+    # print('result finished')
```

Comment:
should probably clean this up.
Consider using Python logging.

Reply:
we do, this was probably some debugging hint